### DISCUSSION TITLE
Support for PUG includes

### DISCUSSION THREAD

**Discussion by silentmantra** [vitejs/vite-plugin-vue#452] [Q&A] [answered]
### Description

Promoting Vite on stackoverflow.com but that's hard. I guess Vite loses to Webpack in terms of support and size of community. I think more easy-to-implement features could be added to Vite for closing the gap.

I had recently a hot debate on handling PUG files.
https://stackoverflow.com/questions/76565215/vite-not-detecting-changes-vue-pug

Turned out there's no support for PUG includes in VIte and a couple of existing PUG plugins doesn't solve the problem.

### Suggested solution

Add support of using `include` inside `<template lang="pug">`

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitejs/vite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitejs.dev/guide).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by rashfael** [comment, marked as answer]
I got pug includes to work in our projects by specifying the basedir in the vue plugin config like this:
```js
vue({
  template: {
    preprocessOptions: {
      basedir: path.resolve(__dirname, './src') // or something else here
    },
  }
}
```
